selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/client/TestGitHubRepository.java

```diff
@@ -56,63 +60,80 @@ private GHRepository create() throws IOException, InterruptedException {
   private void ensureRepositoryCreated(GHRepository repo) throws IOException {
     Throwable lastIOException = null;
 
-    for (int i = 0; i < REPO_CREATION_TIMEOUT_SEC; i++) {
+    long startTime = System.currentTimeMillis();
+    for (int i = 0; i < REPO_CREATION_ATTEMPTS; i++) {
       try {
         gitHub.getRepository(repo.getFullName());
         return;
       } catch (IOException e) {
         lastIOException = e;
         LOG.info("Waiting for {} to be created", repo.getHtmlUrl());
-        sleepQuietly(1); // sleep one second
+        sleepQuietly(GITHUB_OPERATION_TIMEOUT_SEC); // sleep one second
       }
     }
 
     throw new IOException(
         format(
             "GitHub repo %s hasn't been created in %s seconds",
-            repo.getHtmlUrl(), REPO_CREATION_TIMEOUT_SEC),
+            repo.getHtmlUrl(), (System.currentTimeMillis() - startTime) / 1000),
```

Comment:
Would you encapsulate this code part:
```
(System.currentTimeMillis() - startTime) / 1000
```
to the private method for more readibility, something like:
```
int getStartingTime(long startTime)
```

Reply:
If it is not clear enough what that statement does, I would extract it into separate variable.